client/src/styles/globals.css

```diff
@@ -35,3 +35,69 @@
     @apply px-4 py-3;
   }
 }
+
+/* Taken from https://jerrylowm.medium.com/build-a-tags-input-react-component-from-scratch-1524f02acb9a */
+@layer fileTags {
```

Comment:
I'm personally not sure how the @layer tag works here since fileTags isn't a tailwind specific section it would be expection, but looks good. I would say this is fine and it would be perfect to use as practice converting this to tailwind in a separate PR.